packages/suite/src/components/wallet/WalletLayout/components/WalletLayoutNavigation.tsx

```diff
@@ -38,13 +38,13 @@ const NavLinkText = styled.div`
 
 const Badge = styled.span`
     position: absolute;
-    top: -14px;
-    right: -27px;
+    top: -12px;
+    right: -24px;
     padding: 2px 4px 0 4px;
     border-radius: 4px;
     background: ${({ theme }) => theme.BG_LIGHT_GREEN};
     color: ${({ theme }) => theme.TYPE_GREEN};
-    font-size: ${variables.FONT_SIZE.TINY};
+    font-size: 10px;
```

Comment:
Why is this being reverted? I was asked to use a constant instead of hard-coded value in my last PR.

Reply:
I thought 10px was too small. And "Powered by invity" on the left is already big, so it's not like the footer becomes less obtrusive if the right part is 10px ;)